Makefile

```diff
@@ -51,7 +73,14 @@ mysql_build: $(CMD_FILES) $(PKG_FILES)
 	(cd $(MAIN_MYSQL_PATH) && go build -tags brotli -o wal-g -ldflags "-s -w -X github.com/wal-g/wal-g/cmd.BuildDate=`date -u +%Y.%m.%d_%H:%M:%S` -X github.com/wal-g/wal-g/cmd.GitRevision=`git rev-parse --short HEAD` -X github.com/wal-g/wal-g/cmd.WalgVersion=`git tag -l --points-at HEAD`")
 
 mysql_integration_test:
-	docker-compose build $(DOCKER_COMMON) mysql mysql_tests
+	@if [ ! -f ${CACHE_FILE_UBUNTU} ]; then\
+		echo "Rebuild";\
+		make build_image;\
+	else\
+		docker load -i ${CACHE_FILE_UBUNTU};\
+		docker load -i ${CACHE_FILE_GOLANG};\
```

Comment:
Why this part is different from pg _integration_test?

Reply:
Here we should build just DOCKER_COMMON
In pg_integration_test we should build DOCKER_COMMON + pg + docker_prefix